API_DOCUMENTATION.md

```diff
@@ -9,5 +9,48 @@
   **Response:**
   {body: string}
 
+  ---
+
+#### POST "/messages/"
+  *Create a new message with given JSON body information*
+  
+  **Params:**
+  {
+  text: string,
+  senderId: int,
+  receiverId: int
+  }
+
+  **Response:**
+  STATUS 200 OK
+
+#### GET "/messages/conversations/{userId}"
```

Comment:
Change GET "/messages/conversations/{userId}" to GET "/users/{id}/contacts". This endpoint is needed to  display the quick/last message menu for a certain user, so it makes more sense to store it under /users/{id}/…

Reply:
OK, will also move the conversation query into user query manager